scripts/runhttpd

```diff
@@ -25,6 +25,8 @@ else
     INSPECTOR_EXTRA_ARGS=" ipa-inspection-callback-url=${IRONIC_BASE_URL}:5050/v1/continue"
 fi
 
+. /bin/coreos-ipa-common.sh
```

Comment:
This is a race condition. At least in the installer's startironic.sh script, the way to identify whether an image is fully downloaded, there exists a curl loop to check for finished file. Looks like this statement here is meant to add the kernel params, but when the httpd container is started, the images are not yet downloaded and the kernel params will not be set.

I did move the startup of the httpd container around in my installer fork as a test and it worked fine but it still feels racy.

https://github.com/kirankt/installer/blob/222922cab48056f85fb04716cb40591394200882/data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template#L161

Reply:
Don't all containers in a pod wait until initContainers are finished?

Reply:
Ah, they're not initContainers in this case. Still, won't this wait work? https://github.com/kirankt/installer/blob/222922cab48056f85fb04716cb40591394200882/data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template#L152

Reply:
You're right. That should catch it. But in practice the curl loops have been in place to check whether the download is indeed finished. In my testing too, this has been the case.

Reply:
You mean, the curl was required in your case? If so, that's a bug we should fix separately, I guess.